internal/storage/secrets.go

```diff
@@ -146,7 +147,11 @@ func fileExists(path string) (bool, error) {
 
 func promptPassword(prompt string) (string, error) {
 	console.Printf(prompt)
-	b, err := term.ReadPassword(int(os.Stdin.Fd()))
+	intFd, err := safecast.ToInt(uint(os.Stdin.Fd()))
+	if err != nil {
+		return "", err
+	}
+	b, err := term.ReadPassword(intFd)
```

Comment:
I found https://pkg.go.dev/github.com/charmbracelet/x/term which takes a uintptr for `ReadPassword` too